jsapp/js/account/accountFieldsEditor.component.tsx

```diff
@@ -176,32 +219,58 @@ export default function AccountFieldsEditor(props: AccountFieldsEditorProps) {
       </div>
 
       <div className={styles.row}>
-        {/* Organization */}
-        {isFieldToBeDisplayed('organization') && (
-          <div className={styles.field}>
-            <TextBox
-              label={getLabel('organization')}
-              onChange={onAnyFieldChange.bind(onAnyFieldChange, 'organization')}
-              value={props.values.organization}
-              errors={props.errors?.organization}
+        {/* Organization Type */}
+        {isOrganizationTypeFieldToBeDisplayed() && (
```

Comment:
Most fields want to be half-width now so I may use a flex wrap similar to the sign-up form in case the field set is very different.

The quick fix will be to re-pair the rows without changing the CSS. For servers that include 'gender' and 'city' things would have to move around.